Actions/ReadSecrets/ReadSecretsHelper.psm1

```diff
@@ -1,13 +1,27 @@
 $script:gitHubSecrets = $env:secrets | ConvertFrom-Json
 $script:keyvaultConnectionExists = $false
 $script:azureRm210 = $false
-$script:isKeyvaultSet = $script:gitHubSecrets.PSObject.Properties.Name -eq "AZURE_CREDENTIALS"
+$script:isKeyvaultSet = ($script:gitHubSecrets.PSObject.Properties.Name -eq "AZURE_CREDENTIALS") -or ($script:gitHubSecrets.PSObject.Properties.Name -eq "AZURE_KEYVAULT_URI")
 $script:escchars = @(' ','!','\"','#','$','%','\u0026','\u0027','(',')','*','+',',','-','.','/','0','1','2','3','4','5','6','7','8','9',':',';','\u003c','=','\u003e','?','@','A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z','[','\\',']','^','_',[char]96,'a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z','{','|','}','~')
 
 function IsKeyVaultSet {
     return $script:isKeyvaultSet
 }
 
+function Get-KeyVaultName {
+    if ($script:gitHubSecrets.PSObject.Properties.Name -eq "AZURE_KEYVAULT_URI") {
```

Comment:
If people are using a KeyVault for their secrets already - and uptake this feature - then the existing keyvault will stop working as you will always return the name of the keyvault containing the certificate - right?

Reply:
Yes - In this setup we only support one KeyVault. So, people would have to keep their certificates in the same KeyVaults as their other secrets.